sale_order_line_date/README.rst

```diff
@@ -14,12 +14,12 @@ of the lines change to match.
 Usage
 =====
 
-Create a Quotation or a Sales Order and fill the requested date in the sale
+Create a Quotation or a Sales Order and it fills the requested date in the sale
```

Comment:
@darshan-serpent We can discuss if we should change the previous behavior but the previous behavior is exactly at the readme describes. The method def _onchange_requested_date_ is supposed to do that. It's not working properly. :confused: